braille-printer-app/brf-printer.h

```diff
@@ -0,0 +1,498 @@
+    },
+    {
+        "image/x-ms-bmp",
+        "image/vnd.cups-ubrl",
+            {cfFilterExternal, &imagetoubrl_filter, "imagetobrf"}
+    },
+    {
+        "image/x-portable-anymap",
+        "image/vnd.cups-ubrl",
+            {cfFilterExternal, &imagetoubrl_filter, "imagetoubrl"}
+    },
+    {
+        "image/x-portable-bitmap",
+        "image/vnd.cups-ubrl",
+            {cfFilterExternal, &imagetoubrl_filter, "imagetoubrl"}
+    },
+    {
+        "image/x-portable-graymap",
+        "image/vnd.cups-ubrl",
+            {cfFilterExternal, &imagetoubrl_filter, "imagetoubrl"}
+    },
+    {
+        "image/x-portable-pixmap",
+        "image/vnd.cups-ubrl",
+            {cfFilterExternal, &imagetoubrl_filter, "imagetoubrl"}
+    },
+    {
+        "image/x-xbitmap",
+        "image/vnd.cups-ubrl",
+            {cfFilterExternal, &imagetoubrl_filter, "imagetoubrl"}
+    },
+    {
+        "image/x-xpixmap",
+        "image/vnd.cups-ubrl",
+            {cfFilterExternal, &imagetoubrl_filter, "imagetoubrl"}
+    },
+    {
+        "image/x-xwindowdump",
+        "image/vnd.cups-ubrl",
+            {cfFilterExternal, &imagetoubrl_filter, "imagetoubrl"}
+    },
+
+
+    {
+        "image/svg",
+        "image/vnd.cups-pdf",
+            {cfFilterExternal, &svgtopdf_filter, "svgtopdf"}
+    },
+
+    {
+        "image/svg+xml",
+        "image/vnd.cups-pdf",
+            {cfFilterExternal, &svgtopdf_filter, "svgtopdf"}
+    },
+
+    {
+        "application/x-xfig",
+        "image/vnd.cups-pdf",
+            {cfFilterExternal, &xfigtopdf_filter, "xfigtopdf"}
+    },
+
+    {
+        "image/wmf",
+        "image/vnd.cups-pdf",
+            {cfFilterExternal, &wmftopdf_filter, "wmftopdf"}
+    },
+
+    {
+        "image/x-wmf",
+        "image/vnd.cups-pdf",
+            {cfFilterExternal, &wmftopdf_filter, "wmftopdf"}
+    },
+
+    {
+        "windows/metafile",
+        "image/vnd.cups-pdf",
+            {cfFilterExternal, &wmftopdf_filter, "wmftopdf"}
+    },
+    {
+        "application/x-msmetafile",
+        "image/vnd.cups-pdf",
+            {cfFilterExternal, &wmftopdf_filter, "wmftopdf"}
+    },
+    {
+        "image/emf",
+        "image/vnd.cups-pdf",
+            {cfFilterExternal, &emftopdf_filter, "emftopdf"}
+    },
+    {
+        "image/x-emf",
+        "image/vnd.cups-pdf",
+            {cfFilterExternal, &emftopdf_filter, "emftopdf"}
+    },
+    {
+        "image/cgm",
+        "image/vnd.cups-pdf",
+            {cfFilterExternal, &cgmtopdf_filter, "cgmtopdf"}
+    },
+
+    {
+        "image/x-cmx",
+        "image/vnd.cups-pdf",
+            {cfFilterExternal, &cmxtopdf_filter, "cmxtopdf"}
+    },
+
+    {
+        "image/vnd.cups-pdf",
+        "image/vnd.cups-brf",
+            {cfFilterExternal, &vectortobrf_filter, "vectortobrf"}
+    },
+    {
+        "image/vnd.cups-pdf",
+        "image/vnd.cups-ubrl",
+            {cfFilterExternal, &vectortoubrl_filter, "vectortoubrl"}
+    },
+    {
+    NULL
+    }
+};
```

Comment:
A `.h` file shouldn't be defining `static` variables, rather move them to a `.c` file and non-static, and add here an `extern` declaration for them.